### ISSUE TITLE
Bundle the relayer into the node so that we have a seamless startup for using/testing

### ISSUE THREAD

**Issue by drewstone** [hicommonwealth/substrate-rainbow-bridge#13]
This might be naive, so share your thoughts @shekohex, but I think we should bundle everything into the node so that the service starts on node startup. Do you agree/disagree?

**Comment by shekohex** [comment]
yes indeed we can bundle it in the same project, but the key here we need to separate them into two different processes, since the relayer maybe get killed by the OS on OOM and the supervisor (say docker here for example) can restart it again.

so if they are in the same process, the node will be killed too, which is very bad.
so as we make docker images for the node, we could make the relayer a different simpler image too.
